Package.swift

```diff
@@ -0,0 +1,37 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "govuk-mobile-ios-homepage",
+    platforms: [.iOS(.v14)],
+    products: [
+        // Products define the executables and libraries a package produces, making them visible to other packages.
+        .library(
+            name: "govuk-mobile-ios-homepage",
+            targets: ["Homepage"])
+    ],
+//    dependencies: [
+//        .package(url: "https://github.com/govuk-one-login/mobile-ios-common", branch: "main"),
```

Comment:
Should the package.resolved contain these if they have been commented out?

Reply:
I had originally written this `package.swift` file when the work was to build out the functionality in this package. I was leaving it here because I think they will soon be required as soon as the package isn't empty.

Reply:
Yeah, the comments are fine, but the `package.resolved` file thinks they are still being used?

Reply:
As per other comment, I've deleted the file.